layout/_partial/search.ejs

```diff
@@ -22,13 +22,13 @@
                 <div class="media">
                     <% if (post.thumbnailImageUrl) { %>
                     <div class="media-left">
-                        <a class="link-unstyled" href="<%= post.link || post.permalink %>">
+                        <a class="link-unstyled" href="<%= post.permalink || post.permalink %>">
```

Comment:
tks for PR, but what does `post.permalink || post.permalink` mean, it just can be `post.permalink || post.path` or `post.permalink`